src/components/agents/Agent.jsx

```diff
@@ -28,6 +30,11 @@ export default class Agent extends Component {
       .then(agent => this.setState({ agent }));
   }
 
+  createInteractionRoute(e) {
```

Comment:
Can we rename this something like goToInteraction that better describes what it's doing?